docs/corectl.md

```diff
@@ -14,6 +14,7 @@ corectl [flags]
 
 ```
   -h, --help      help for corectl
+  -t, --traffic   Log JSON traffic to stdout
```

Comment:
QIX WS traffic?

Reply:
In my opinion we shouldn't use `QIX`, but maybe `Log JSON websocket traffic to stdout`?